apps/ui/src/views/Bridge/components/BridgeHistory/ExpandRowContent.tsx

```diff
@@ -40,9 +42,14 @@ export const ExpandRowContent: React.FC<ExpandRowContentProps> = (props) => {
   return (
     <div>
       <div>
-        <TransactionLink network={record.txSummary.fromAsset.network} txId={record.txSummary.fromTransaction.txId}>
-          {fromTransactionDescription}
-        </TransactionLink>
+        <Space>
+          <TransactionLink network={record.txSummary.fromAsset.network} txId={record.txSummary.fromTransaction.txId}>
+            {fromTransactionDescription}
+          </TransactionLink>
+          {confirmStatus === ' (pending)' && record.txSummary.fromAsset.network === 'Nervos' && (
```

Comment:
a ui display string is not recommended to check the status, maybe we should use the `record.status && confirmNumber === 0 ` to instead of

Reply:
fixed